en/products/relay/faq.ftl

```diff
@@ -0,0 +1,144 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/relay/faq/
+
+## FAQ Page
+
+faq-headline = Frequently Asked Questions
+faq-question-what-is-question-2 = What is a { -brand-name-relay } email mask?
+faq-question-what-is-answer-2 = Email masks are masked, or private, email addresses that forward messages to your true email address. These masks allow you to share an address with third parties which will mask your true email address and forward messages to it.
+faq-question-missing-emails-question-2 = I’m not getting messages from my email masks
+faq-question-missing-emails-answer-a-2 = There are a few reasons you might not be receiving emails forwarded through your masks. These reasons include:
+faq-question-missing-emails-answer-reason-spam = Messages are going into spam
+faq-question-missing-emails-answer-reason-blocked-2 = Your email provider is blocking your email masks
+faq-question-missing-emails-answer-reason-size = The email forwarded has an attachment larger than { email-size-limit }
```

Comment:
This is calling a Fluent string that isn't being migrated...
That being said the way that string is formatted is problematic for localization:
```
# String used to display the attachment limit, e.g. 10 MB
# Variables:
#  $size (number): maximum size for attachments
#  $unit (string): unit of measurement (e.g. MB for Megabyte)
email-size-limit = { $size } { $unit }
```
Unit size should be localized, since different languages have different methods of displaying unit sizes (e.g. `KB` is `kB` in Italian and `КБ` in Russian).

Reply:
It will cause repetitions, but I think it would be safer to have the two variables directly in the message.

```
faq-question-missing-emails-answer-reason-size = The email forwarded has an attachment larger than { $size } { $unit }
```

Reply:
You have a point, in that case 

```
faq-question-missing-emails-answer-reason-size = The email forwarded has an attachment larger than { $size } KB
```

Although I suspect the limit is in MB?

Reply:
It started off to be in KB and later the size had increased. Let me see if I can get some one to answer this.

Reply:
The[ current limit is 10MB](https://github.com/search?q=repo%3Amozilla%2Ffx-private-relay%20emailSizeLimitNumber&type=code) I'll update it.

Reply:
And I really doubt the limit will go to GB ;) But if it does we can trigger a re-translation.